src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -250,33 +250,36 @@ def lockout_spiders
 
   def check_user
```

Comment:
On Donnerstag, 29. Oktober 2015, 03:23:31 CET wrote Henne Vogelsang:
> > @@ -250,33 +250,36 @@ def lockout_spiders
> >  
> >    def check_user
> 
> An empty HTTP_X_USERNAME (how can this happen?) is easily checked with .blank in line 259. No need to rewrite the whole logic IMHO...

with a broken proxy (setup), but it was just an example.

I just dislike this if-else-end mess with follow up code. It increases the condition depth without need.

Also the old code creating different conditions depending on the meanwhile obsolete :simulate mode
could be removed that way.